src/model/codacyInput.ts

```diff
@@ -0,0 +1,41 @@
+export type ParameterValue = any
+
+export class Parameter {
```

Comment:
I was expecting a file per model class.
Why do we keep them all in the same file?

Reply:
I grouped classes by "topic" to not have too many files.